server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/main/java/org/apache/james/webadmin/integration/vault/LinshareBlobExportMechanismIntegrationTest.java

```diff
@@ -287,7 +228,7 @@ private void bartSendMessageToHomer() {
             .header("Authorization", bartAccessToken.asString())
             .body(requestBody)
             .post("/jmap")
-        .then()
+            .then()
```

Comment:
Okay, we are quite rigide on REST testing ident...

Then pattern is :

```
given()
    .blah()
    .bloh()
.when() // at the same level than given()
    .blih() // 4 space ident
.then() // at the same level than given()
    .bluh();
```

This convention allow faster reading of REST assertions once you get the habbit of it.

Can you restore this indent?

Reply:
Yes, i will